core/src/main/scala/org/bitcoins/core/protocol/ln/LnInvoice.scala

```diff
@@ -99,7 +99,7 @@ sealed abstract class LnInvoice {
   override def toString: String = {
     val b = new StringBuilder
     b.append(hrp.toString)
-    b.append(Bech32.separator)
+    b.append('1')
```

Comment:
I don't know why i had to use the hard coded `1` but the initilization problem persisted if I didn't add it. When invoices were being serialized in some cases the separator would be missing